asn1c_combined/semi-asn-files/2024/SEMI_v2.4.2_050925_j2024.asn

```diff
@@ -441,13 +442,13 @@ AdvisoryDetails ::= SEQUENCE {
 	distType		DistributionType,
 	startTime		DFullTime	OPTIONAL,
 	stopTime		DFullTime	OPTIONAL,
-	advisoryMessage	OCTET STRING (SIZE(0..1400))	-- Encoded advisory message
+	advisoryMessage	OCTET STRING (SIZE(0..7000))	-- Encoded advisory message
 }
 
 AdvisoryBroadcast ::= SEQUENCE {
 	messagePsid		Psid,								-- PSID of advisory message
 	broadcastInst	BroadcastInstructions OPTIONAL,		-- Broadcast instructions
-	advisoryMessage	OCTET STRING (SIZE(0..1400))		-- Encoded advisory message
+	advisoryMessage	OCTET STRING (SIZE(0..7000))		-- Encoded advisory message

```

Comment:
Note: It was not necessary to modify this constraint to get the new unit tests to pass, but this constraint appears to be coupled with the constraint above as both were set to 1400 and both were for an "Encoded advisory message"